generators/utilityscripts/main/deploy/deploy.ps1

```diff
@@ -1,10 +1,21 @@
-function Uninstall() {
+function Uninstall()
+{
     Write-Host "Uninstalling App as installation failed... Please try installation again."
     Invoke-Expression "& $PSScriptRoot\uninstall.ps1"
     Exit
 }
-
-$AppPath = "$PSScriptRoot\<%= appPackage %>"
+$path="./vscode-config.json"
```

Comment:
this name should be  <%= VSCodeConfigName %> and should be updated by the generator at the time of deploy script creation.